src/pages/Flavors.test.js

```diff
@@ -139,126 +152,92 @@ describe('Page <Flavors />', () => {
       }
     }
   ];
-  const pager = { count: 5, limit: 20, page: 1, pages: 1 };
-  const store = mockStore({
-    actions,
-    application: appInitialState,
-    flavor: flavorInitialState,
-    flavors: flavorsInitialState
-  });
 
+  const pager = { count: 5, limit: 20, page: 1, pages: 1 };
   const pagerNavigation = <p></p>;
 
-  const notLoggedInProps = {
-    actions,
+  const props = {
     collection,
-    loggedIn: false,
     pager,
-    pagerNavigation,
-    stash,
-    stashLoaded: false
+    pagerNavigation
   };
 
-  const noStashProps = {
-    actions,
-    collection,
-    loggedIn: true,
-    pager,
-    pagerNavigation,
-    stash,
-    stashLoaded: false
-  };
+  const ReduxConnectedFlavors = (flavorsProps) => (
+    <Provider store={store}>
```

Comment:
We have a withProvider() utility in src/utils/testing.js that provides a way to do this